src/psyclone/tests/test_files/dynamo0p3/simple_with_scalars.f90

```diff
@@ -42,10 +42,11 @@ module simple_with_scalars_mod
   implicit none
 
   type, extends(kernel_type) :: simple_with_scalars_type
-    type(arg_type), dimension(3) :: meta_args =          &
+    type(arg_type), dimension(4) :: meta_args =          &
```

Comment:
Not yours but comparing this test kernel with `testkern_three_scalars_mod.f90` does beg the question, "why do we have both of them?" It looks as though just one will do. Unless there's a quick answer to this, I suggest opening a new issue to look at this.

Reply:
Related to this, how come no corresponding algorithm file has been modified?

Reply:
As far as I can see, this kernel is only used in the scalar kernel stub tests and nowhere else - so there is no corresponding algorithm. I think that once upon a time it was used in user guide but this is no longer the case (I grepped for all instances). I am happy to remove it in the next review of code iteration and just use `testkern_three_scalars_mod.f90` instead.

Reply:
That would be great, thanks Iva.

Reply:
Removed the kernel and updated the test to use `testkern_three_scalars_mod.f90`.